src/test/resources/application-test.properties

```diff
@@ -22,7 +22,7 @@ spring.datasource.driverClassName = org.h2.Driver
 spring.h2.console.enabled=true
 spring.h2.console.path=/console/
 
-spring.jpa.hibernate.ddl-auto=create-drop
+spring.jpa.hibernate.ddl-auto=update
```

Comment:
Could not having create-drop lead to the older data lying around after the tests?

Reply:
This was actually me who asked April to change it but was only for testing something. It can be switched back to `create-drop`